.github/scripts/get_system_info.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+#  Copyright (C) 2023 Intel Corporation
+#  Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+#  See LICENSE.TXT
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# get_system_info.sh - Script for printing system info
+
+function system_info {
+	echo "**********system_info**********"
+	cat /etc/os-release | grep -oP "PRETTY_NAME=\K.*"
+	cat /proc/version
+	echo "**********SYCL-LS**********"
+	source /opt/intel/oneapi/setvars.sh
+	sycl-ls
+	echo "**********VGA**********"
+	lspci | grep VGA
+	echo "**********CUDA Version**********"
+	if command -v nvidia-smi &> /dev/null; then
+		nvidia-smi
+	else
+		echo "CUDA not installed"
+	fi
+	echo "**********L0 Version**********"
+	# Instructions below will return some minor errors, as they are dependent on the Linux distribution.
+	dpkg -l | grep level-zero || rpm -qa | grep level-zero || zypper se level-zero || echo "level-zero not installed"
```

Comment:
instead of always calling dpkg/rpm/zypper/apt/yum, can you create a wrapper to find which one exists and use that?
Right now you get things like:
```
.github/scripts/get_system_info.sh: line 27: rpm: command not found
.github/scripts/get_system_info.sh: line 27: zypper: command not found
```

At the very least we should suppress these errors.

Reply:
Done